be/src/vec/exec/scan/vfile_scanner.cpp

```diff
@@ -969,16 +968,11 @@ Status VFileScanner::_generate_fill_columns() {
                 return Status::InternalError("failed to find default value expr for slot: {}",
                                              slot_desc->col_name());
             }
-            _missing_columns->emplace(slot_desc->col_name(), it->second);
+            _missing_col_descs.emplace(slot_desc->col_name(), it->second);
         }
     }
 
-    RETURN_IF_ERROR(_cur_reader->set_fill_columns(*_partition_columns, *_missing_columns));
-    if (_cur_reader->fill_all_columns()) {
```

Comment:
Why remove this?

Reply:
Because adding this will make the logic even more confusing. There's no need for cleaning the `_partition_columns` and `_partition_columns `.